scavenger-frontend/src/message.js

```diff
@@ -269,6 +274,11 @@ export default {
           "delete-title": "애플리케이션({0})을 삭제하시겠습니까? jvm, 스냅샷, 호출 기록 등 연관된 모든 데이터가 삭제됩니다.",
           "delete-success": "애플리케이션을 삭제하였습니다."
         }
+      },
+      export: {
+        "export": "내보내기",
+        "method-invocation": "메소드 호출",
```

Comment:
```suggestion
        "method-invocation": "메서드 호출",
```
As other sentences of the project refer "method" as "메서드", I think it would be nice to make it consistent.

Reply:
Fixed it on commit 3b49bab.